### ISSUE TITLE
models differences

### ISSUE THREAD

**Issue by PrielHaz** [HongguLiu/Deepfake-Detection#33]
what is the difference between:
ffpp_c40.pth
ffpp_c23.pth
deepfake_c0_xception.pkl

All of them are exceptionNet?
The difference is the dataset they trained on?

**Comment by HongguLiu** [comment]
Yes, all of them are based on XceptionNet and the differences are the training datasets.
